src/Psalm/Internal/Cli/Psalm.php

```diff
@@ -272,7 +272,7 @@ public static function run(array $argv): void
 
         self::emitMacPcreWarning($options, $threads);
 
-        self::restart($options, $threads, $progress);
+        //self::restart($options, $threads, $progress);
```

Comment:
that should stay :)

Reply:
Yes it should, forgot to exclude that (it's how I get psalm to run with Xdebug, do let me know if there is a better way). Now undone